tests/getBias.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
No.  We were converting all these to say python2 at some point, but didn't follow through.  We don't even run the tests that way usually.

I asked Russell about this, and he suggested just leaving it alone until we decide what to do with these in general.